tests/IntegrationTests/Attributes.cs

```diff
@@ -12,6 +12,11 @@ public SkippableFactAttribute(ServerFeatures serverFeatures)
 	{
 	}
 
+	public SkippableFactAttribute(ServerFeatures[] serverFeatureList)
```

Comment:
`ServerFeatures` is a `[Flags]` enum, so we can use bitwise OR here to specify more than one feature.